terms.c

```diff
@@ -1662,24 +1662,26 @@ graphend(void)
 int
 graph_ok(void)
 {
-    if (UseGraphicChar != GRAPHIC_CHAR_DEC)
-	return 0;
+    if (UseGraphicChar != GRAPHIC_CHAR_DEC) {
+        return 0;
+    }
     return T_as[0] != 0 && T_ae[0] != 0 && T_ac[0] != 0;
```

Comment:
From a first quick glance: Can you remove this style-only changes? Not using braces here is the dominant style, I prefer to have it consistent.

Reply:
Done. Personally I'd suggest to change the coding style in order to gain readability, but its true it should be consistent.